### DISCUSSION TITLE
IOS : Ionic @capacitor-community/bluetooth-le issue during connection with multiple device

### DISCUSSION THREAD

**Discussion by chandan1726** [capacitor-community/bluetooth-le#729] [Q&A]
IOS APP developed in Ionic v7, when using the @capacitor-community/bluetooth-le plugin, connecting to a second Bluetooth device in IOS Phone simultaneously causes the first device to disconnect.

**Comment by peitschie** [comment]
Hi @chandan1726 

Are you able to share your relevant code snippets and Xcode debug logs here?

The plugin should support this use-case just fine, so there might be something unexpected about your usecase here that the plugin doesn't correctly handle.
